Cargo.toml

```diff
@@ -18,8 +18,11 @@ members = [
 edition = "2021"
 version = "0.0.1"
 license = "MIT OR Apache-2.0"
-# homepage = "https://vello.dev" - Domain owned by us, but unused at present
-# rust-version = 
+homepage = "https://vello.dev"
```

Comment:
Similarly to Peniko, I'd just remove this for now.

Reply:
Since it at least goes somewhere, I think it's fine for now. If we publish it and later there's a real homepage there, it'll be better that it had been there.

Reply:
I'll remove this for now, since it is controversial.